slides/src/content/component.md

```diff
@@ -112,13 +112,26 @@ _src/app/app.component.html_
 
 - Whoops: compiler error!
 
-_src/app/app.component.html_ (with error)
+_src/app/app.component.html_
 ```html
 <app-to-do-list [heading]="'Things To Do'"></app-to-do-list>
 ```
 
 - Have to quote the value inside the double quotes or Angular will try to evaluate it
 
+
+---
+
+## Passing Data to a Component
+
+- Angular also has a shorthand for passing string properties to components:
```

Comment:
Is this actually an Angular thing, or is it just regular old HTML kicking in because there isn't a `[]` or `()` around the attribute name?